src/wizard/Step_1/components/ViewPortfolio.vue

```diff
@@ -1,11 +1,131 @@
+    </div>
+    <v-row>
+      <v-col offset="1">
+        <v-card width="30rem" class="mt-5">
+          <v-card-title class="h3 font-weight-bold">Portfolio ></v-card-title>
+          <v-card-subtitle>portfolio description</v-card-subtitle>
+          <v-card-text>portfolio managers</v-card-text>
+          <v-card-actions class="d-flex">
+            <a
+              class="font-weight-bold h6 text-decoration-underline mt-1 mx-1"
+              href="#"
+              >Open</a
+            >
+            <a
+              class="
+                font-weight-bold
+                h6
+                text-decoration-underline
+                mt-1
+                mx-1
+                ml-auto
+              "
+              href="#"
+              >Delete</a
+            >
+          </v-card-actions>
+        </v-card>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
 
 <script lang="ts">
 import Vue from "vue";
 import { Component } from "vue-property-decorator";
+import { Nav } from "@/components/ATATHeaderNav.vue";
+export interface Portfolio {
+  id: string;
+  name: string;
+  description: string;
+  csp_provisioning_status: string;
+  dod_component: string[];
+  portfolio_managers: string[];
+  applications: Application[];
+}
+export interface Application {
+  id: string;
+  name: string;
+  description: string;
+  environments: Environments[];
+}
+export interface Environments {
+  id: string;
+  name: string;
+  funding_source: string;
+}
+
+const donePortfolio: Portfolio = {
+  id: "12",
+  name: "Defense Logistics Agency",
+  description:
+    "This Portfolio will be used to build,test and manage the native applications for the defense logistics agency",
+  csp_provisioning_status: "complete",
+  dod_component: ["Air Force"],
+  portfolio_managers: ["joe.manager@example.com"],
+  applications: [
+    {
+      id: "74",
+      name: "APPLICATION_NAME",
+      description: "APPLICATION_DESCRIPTION",
+      environments: [
+        {
+          id: "765d364f-d7f1-4f28-be8f-123d915b68bb",
+          name: "ENVIRONMENT_NAME",
+          funding_source: "599948c4-35c1-49ee-a7ab-9190ca2c1a81",
+        },
+      ],
+    },
+  ],
+};
+const draftPortfolio: Portfolio = {
+  id: "13",
+  name: "Defense  Agency",
+  description:
+    "This Portfolio will be used to build,test and manage the native applications for the defense logistics agency",
+  csp_provisioning_status: "draft",
+  dod_component: ["Army"],
+  portfolio_managers: ["joe.manager@example.com"],
+  applications: [
+    {
+      id: "75",
+      name: "APPLICATION_NAME",
+      description: "APPLICATION_DESCRIPTION",
+      environments: [
+        {
+          id: "765d364f-d7f1-4f28-be8f-123d915b68bb",
+          name: "ENVIRONMENT_NAME",
+          funding_source: "599948c4-35c1-49ee-a7ab-9190ca2c1a81",
+        },
+      ],
+    },
+  ],
+};
 
 @Component({})
 export default class ViewPortfolio extends Vue {}
 </script>
 
-<style scoped></style>
+<style scoped>
```

Comment:
Create this file  `src\sass\components\_v-card.scss` and move all sass code to this file.
Add a reference to this file in 'src\sass\atat.scss'